### PR TITLE
add docker network documentation

### PR DESCRIPTION
**Pull request by GuyAfik**
## Status
Ready

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-5505)

## Description
Add docker network docs

### CODE REVIEW

File: docs/integrations/unit-testing.md
Comment (ShirleyDenkberg) [inline docs/integrations/unit-testing.md:61]:
```suggestion
By default, unit-tests are not running with a Docker network, meaning the network is disabled within the container that runs the unit-tests.
```

File: docs/packs/packs-format.md
Comment (ShirleyDenkberg) [inline docs/packs/packs-format.md:166]:
```suggestion
* `linter code` e.g., IN126 - ignore linter error codes.
```

File: docs/packs/packs-format.md
Comment (ShirleyDenkberg) [inline docs/packs/packs-format.md:168]:
```suggestion
2) By default, unit-tests of scripts/integrations are running without a docker network.
```

File: docs/packs/packs-format.md
Comment (ShirleyDenkberg) [inline docs/packs/packs-format.md:170]:
```suggestion
   In case one of the integrations/scripts inside a pack needs a network during the unit-tests run, this can be done in this format
```

**Comment by ShirleyDenkberg** [comment]
@GuyAfik Doc review completed.

File: docs/integrations/unit-testing.md
Comment (JudahSchwartz) [inline docs/integrations/unit-testing.md:61]:
```suggestion
By default, unit-tests are not running with access to the network; the network is disabled within the container that runs the unit-tests.
```

File: docs/integrations/unit-testing.md
Comment (JudahSchwartz) [inline docs/integrations/unit-testing.md:63]:
```suggestion
Refer to [here](../packs/packs-format.md#pack-ignore) if the script/integration tests need access to the network. 
```

**Review by JudahSchwartz — APPROVED** [review]

**Comment by JudahSchwartz** [comment]
lets not merge until the feature is merged

**Comment by GuyAfik** [comment]
blocked at the moment, will be merged soon.

**Comment by GuyAfik** [comment]
waiting for a call

**Comment by GuyAfik** [comment]
will be merged soon

File: docs/integrations/unit-testing.md
Comment (JudahSchwartz) [inline docs/integrations/unit-testing.md:61]:
```suggestion
By default, unit-tests are not run with access to the network; the network is disabled within the container that runs the unit-tests.
```

**Review by JudahSchwartz — APPROVED** [review]
nice

**Comment by GuyAfik** [comment]
Will be merged when the new SDK version will be released.

**Pull request merged**
